lldb/test/API/functionalities/scripted_process/missing_methods_scripted_process.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+
+class MissingMethodsScriptedProcess:
+    def __init__(self, exe_ctx, args):
+        pass
+
+
+def __lldb_init_module(debugger, dict):
+    if not "SKIP_SCRIPTED_PROCESS_LAUNCH" in os.environ:
```

Comment:
Instead of doing this through an environment variable, is there a way we can do this with the `dict` or as a `debugger` setting?

Reply:
Not really, this is how we've been doing it for all the ScriptedProcess tests.